spec/features/admin/index_spec.rb

```diff
@@ -98,17 +98,17 @@
       @invoice_item_6 = create(:invoice_item, status: "shipped", item: item_6, invoice: @invoice_3)
     end
 
-    it "has a section for Incomplete Invoices that displays linked ids that have not been shipped" do
+    it "has a section for Incomplete Invoices that displays linked ids that have not been shipped, and shows when they were created at, ordered oldest to newest" do
```

Comment:
This is a little lengthy.
```suggestion
    it "has a section for unshipped Incomplete Invoices with ids as links, ordered by creation, oldest to newest" do
```

Reply:
Love it, updated